app/core/Router.coffee

```diff
@@ -115,6 +115,8 @@ module.exports = class CocoRouter extends Backbone.Router
     'editor/thang-tasks': go('editor/ThangTasksView')
     'editor/verifier': go('editor/verifier/VerifierView')
     'editor/verifier/:levelID': go('editor/verifier/VerifierView')
+    'editor/i18n-verifier/:levelID': go('editor/verifier/i18nVerifierView')
+    'editor/i18n-verifier': go('editor/verifier/i18nVerifierView')
```

Comment:
Its only purpose is to show us how much of the top error messages for a given level(s) are translated into a given language. Mostly useful for making sure messages are included in `en.coffee` to be translated.